openedx/adg/lms/webinars/helpers.py

```diff
@@ -339,3 +341,14 @@ def get_webinar_invitees_emails(webinar_form):
         )
 
     return webinar_invitees_emails
+
+
+def emit_webinar_registration_event(user_id, topic):
```

Comment:
Since we are creating a separate `analytics.py` module for other applications, for consistency we should do the same for webinars. It would also ensure that if we have to introduce some new analytics related functions for webinars in the future, we would be able to add it in the analytics file.

Reply:
I didn't make one since there was only a single function for it. Adding a separate file